### PR TITLE
Document comma separated attributes in the search service panel

### PR DESCRIPTION
**Pull request by chpicone**
## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: documentation (clearify and some corrections)

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

It adds exaplanation about how to set more than one attributes (comma separated) when the user want to create a new search service.
It also corrects a translation error in the IT language in the Catalog planel (Impostazioni avanzante => Impostazioni avanzate)

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/user-guide/menu-bar.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ In particular, the information to be entered are:
 
 * **Layer** to be queried
 
-* Specific **Attributes** (fileds) the user wants to query
+* Specific **Attributes** (comma separeted fields) the user wants to query
```
Comment (MV88) [inline docs/user-guide/menu-bar.md:68]: Hi, i think we need to correct the old and the new typo 😄 
```suggestion
* Specific **Attributes** (comma separated fields) the user wants to query
```

**Comment by MV88** [comment]
Hi @chpicone,
please do a backport to the stable branch 2020.02xx

**Comment by tdipisa** [comment]
@chpicone please backport this PR to 2020.02.xx for the release. thank you.

**Comment by chpicone** [comment]
@tdipisa  PR is done [here](https://github.com/geosolutions-it/MapStore2/pull/5446)

**Pull request merged**